apps/frontend/src/pages/setting/index.module.scss

```
@@ -21,4 +54,75 @@
 
 .setting-divider {
     margin: 32px 0 24px;
+    border-color: var(--color-border) !important;
+}
+
+/* 表单样式覆盖 */
+:global(.ant-form-item-label) {
+    label {
+        font-family: 'Geist', 'Inter', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)